db/seeds.rb

```diff
@@ -8,6 +8,7 @@
                     fetch(Rails.env, {})
 
 service_providers.each do |issuer, config|
+  next if Figaro.env.chef_env == 'prod' && config['live'] != 'true'
```

Comment:
Maybe the name `live` for the key is contributing to the confusion? What if we called this `chef_env`?

Then we could do `Figaro.env.chef_env == config['chef_env']` or something and that would let us explicitly hardcode things for things like `int` as well?